src/System.Private.CoreLib/src/System/Collections/Generic/ArraySortHelper.CoreRT.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Diagnostics;
+using System.Runtime.Versioning;
+
+namespace System.Collections.Generic
+{
+    internal partial class ArraySortHelper<T>
+    {
+        // WARNING: We allow diagnostic tools to directly inspect this member (s_defaultArraySortHelper). 
```

Comment:
@jkotas MembersMustExist has a dependency on the `s_defaultArraySortHelper` being exactly what it was originally. Should we update MembersMustExist, or rollback the change to how this field is defined?

Cc @cshung